Automattic/jetpack
# Boost: fix flashing Failed to generate CSS message

#28714 introduced a bug where the error message disappears if the success count is 0

### Other information:

## Jetpack product discussion
N/A

## Does this pull request change what data or activity we track or use?
N/A

## Testing instructions:
- Apply patch
- Generate CSS and make the front-end die to produce 0 css files generated
- Ensure that the error message doesn't flash
